### ISSUE TITLE
Failed to build

### ISSUE THREAD

**Issue by 4ntoine** [avr-llvm/clang#13]
```
[ 85%] Building CXX object tools/clang/lib/CodeGen/CMakeFiles/clangCodeGen.dir/BackendUtil.cpp.o
/Volumes/Transcend/dev/src/llvm-avr/llvm/tools/clang/lib/CodeGen/BackendUtil.cpp:732:42: error: expected class name
class ThinLTOOutputWrapper : public lto::NativeObjectOutput {
                                         ^
/Volumes/Transcend/dev/src/llvm-avr/llvm/tools/clang/lib/CodeGen/BackendUtil.cpp:738:50: error: only virtual member
      functions can be marked 'override'
  std::unique_ptr<raw_pwrite_stream> getStream() override {
                                                 ^~~~~~~~~
/Volumes/Transcend/dev/src/llvm-avr/llvm/tools/clang/lib/CodeGen/BackendUtil.cpp:789:17: error: no matching function
      for call to 'thinBackend'
  if (Error E = thinBackend(
                ^~~~~~~~~~~
/Volumes/Transcend/dev/src/llvm-avr/llvm/include/llvm/LTO/LTOBackend.h:42:7: note: candidate function not viable: no
      known conversion from '(lambda at
      /Volumes/Transcend/dev/src/llvm-avr/llvm/tools/clang/lib/CodeGen/BackendUtil.cpp:785:20)' to 'AddStreamFn' (aka
      'function<std::unique_ptr<NativeObjectStream> (unsigned int)>') for 3rd argument
Error thinBackend(Config &C, unsigned Task, AddStreamFn AddStream, Module &M,
      ^
```

```
MBA-Anton:build asmirnov$ cd ../llvm/tools/clang/
MBA-Anton:clang asmirnov$ git log -1
commit 0cca6076b017ec2f323d570e658d9efcc9dce9b9
Merge: ee44fb6 236cf68
Author: Dylan McKay <dylanmckay34@gmail.com>
Date:   Thu Sep 15 22:15:18 2016 +1200

    Merge branch 'master' into avr-support

MBA-Anton:clang asmirnov$ cd ../..
MBA-Anton:llvm asmirnov$ git log -1
commit 3c0d2adc4d75f34bbeb3639172197b837409bedf
Author: Dylan McKay <dylanmckay34@gmail.com>
Date:   Sun Sep 25 23:43:24 2016 +1300

    Support upper-case pointer registers in inline assembly
```

**Comment by dylanmckay** [comment]
Fixed in f9591adbe0f75aa58f733f5c1bb99b9e87302d05

I had forgotten to update `clang` when I last updated LLVM

**Comment by 4ntoine** [comment]
Thanks
